content/configuration/configuration_reference.md

```diff
@@ -699,12 +717,91 @@ Two users would be in the role 'pipeline-operators', they are **Jez** and **lqia
 </role>
 ```
 
+
+## &lt;policy&gt; {#policyinrole}
+
+The `<policy>` element defines the permissions for the users belonging to the users of a role.
+
+In a `<policy>` element, the *allow* permission can added by adding the sub-tag [`<allow>`](#allow_permission_in_policy) and *deny* permission can be added by adding the sub-tag [`<deny>`](#deny_permission_in_policy).
+
+**Examples**
+
+The following policy would grant users the permissions to `administer` to all environments expect environments matching `prod_*` name.
+
+```xml
+<policy>
+    <deny action="administer" type="environment">prod_*</deny>
+    <allow action="administer" type="environment">*</allow>
+</policy>
+```
+
+## &lt;allow&gt; {#allow_permission_in_policy}
+
+The `<allow>` element under `policy` grants permissions to the defined GoCD entity based on the type, action and resource.
+
+The value of the `allow` element defines the resource on which policy should be applied. 
+
+ **Notes:**
+  
+| Attribute | Required | Description |
+|-----------|----------|-------------|
+| type      | Yes      | The type of the GoCD entity to grant permissions on. The supported values for type are `environment` and `config_repo`. |
```

Comment:
How about, instead of `The type of the GoCD entity` - we just say `The GoCD entity`?